### ISSUE TITLE
LogLevel::NO_LOG cannot be used

### ISSUE THREAD

**Issue by samijnih-hermes** [configcat/php-sdk#27]
### Describe the bug

We want setup our log level to `LogLevel::NO_LOG` but that value is not handled by `ConfigCat\Log\LogLevel::isValid()` which is called at https://github.com/configcat/php-sdk/blob/master/src/ConfigCatClient.php#L92

### To reproduce

```php
<?php

$configCatClient = new ConfigCatClient('foo', [
    'log-level' => LogLevel::NO_LOG,
    'flag-overrides' => new FlagOverrides(
        new ArrayDataSource([]),
            OverrideBehaviour::LOCAL_ONLY,
        ),
]);

$this->configCatClient->getValue('foo', false);
```

It will actually log something like `[2022-11-16T10:45:20.719634+01:00] ConfigCat.ERROR: Config JSON is not present. Returning defaultValue: '1'.`.

### Expected behavior

We are expecting not to get a log

**Comment by z4kn4fein** [comment]
Hi @samijnih-hermes, thank you for reporting this issue! The fix is released in `v6.2.1`, could you please confirm that it's working now as you expected? Thanks!

**Comment by samijnih-hermes** [comment]
Hi @z4kn4fein , thanks for your quick response. I confirm it's working now 👌 

Have a good day!
